corehq/ex-submodules/dimagi/utils/django/email.py

```diff
@@ -28,7 +28,7 @@
 def mark_local_bounced_email(bounced_addresses, message_id):
     from corehq.apps.sms.models import MessagingEvent, MessagingSubEvent
     from corehq.apps.users.models import Invitation, InvitationStatus
-    if Invitation.EMAIL_ID_PREFIX in message_id:
+    if isinstance(message_id, str) and Invitation.EMAIL_ID_PREFIX in message_id:
```

Comment:
I think since it was introduced, this always caused the following error regularly: https://sentry.io/organizations/dimagi/issues/1899628104/?environment=production&project=136860&query=is%3Aunresolved

> argument of type 'int' is not iterable

Adding this fix to this PR